sources/packages/web/src/views/aest/institution/Restrictions.vue

```diff
@@ -1,14 +1,215 @@
 <template>
   <v-card class="mt-4">
     <div class="mx-5 py-4">
-      <p class="category-header-large color-blue">Restrictions</p>
+      <content-group>
+        <v-row class="m-2">
+          <v-col class="category-header-medium color-blue"
+            >All Restrictions</v-col
+          >
+          <v-col
+            ><v-btn
+              @click="addInstitutionRestriction"
+              class="float-right primary-btn-background"
+              ><font-awesome-icon :icon="['fas', 'plus']" class="mr-2" />Add
+              restriction</v-btn
+            ></v-col
+          >
+        </v-row>
+        <DataTable
+          :value="institutionRestrictions"
+          :paginator="true"
+          :rows="DEFAULT_PAGE_LIMIT"
+          :rowsPerPageOptions="PAGINATION_LIST"
+        >
+          <template #empty>
+            <p class="text-center font-weight-bold">No records found.</p>
+          </template>
+          <Column
+            field="restrictionCategory"
+            header="Category"
+            sortable="true"
+          ></Column>
+          <Column field="description" header="Reason"></Column>
+          <Column field="createdAt" header="Added"
+            ><template #body="slotProps">{{
+              dateOnlyLongString(slotProps.data.createdAt)
+            }}</template></Column
+          >
+          <Column field="updatedAt" header="Resolved">
+            <template #body="slotProps">{{
+              slotProps.data.isActive
+                ? "-"
+                : dateOnlyLongString(slotProps.data.updatedAt)
+            }}</template></Column
+          >
+          <Column field="isActive" header="Status">
+            <template #body="slotProps">
+              <StatusBadge
+                :status="
+                  slotProps.data.isActive
+                    ? GeneralStatusForBadge.ActiveRestriction
+                    : GeneralStatusForBadge.ResolvedRestriction
+                "
+              />
+            </template>
+          </Column>
+          <!-- TODO: the color attribute has to come from either global constant or styling needs to be added to added. -->
+          <Column field="restrictionId" header="">
+            <template #body="slotProps">
+              <v-btn
+                color="#2965c5"
```

Comment:
may be you can declare this as a const in the file and use here

Reply:
I did not add this to a constant as we have a todo for this which will make this value as a global constant.

TODO: the color attribute has to come from either global constant or styling needs to be added to added.